.golangci.yml

```diff
@@ -25,6 +22,7 @@ linters:
   disable:
     - scopelint         # too many false positives
     - gochecknoglobals  # mostly useless
+    - unused            # too many false positives
```

Comment:
I have an idea of how to fix those warnings and will try it soon. Let's revert this change now.